awx/main/models/workflow.py

```diff
@@ -676,3 +681,9 @@ def deny(self, request=None):
         self.save()
         schedule_task_manager()
         return reverse('api:workflow_approval_deny', kwargs={'pk': self.pk}, request=request)
+
+    # &&&&&& Possible placeholder for websocket support
+    # def websocket_emit_data(self):
+    #     websocket_data = super(WorkflowApproval, self).websocket_emit_data()
+    #     websocket_data.update(dict(project_id=self.project.id))  # ?????
```

Comment:
If we're just emitting this for the purposes of detecting new pending approvals, I dunno that there's a lot of *additional* metadata we need to attach to the event we ship out (i.e., we may not need this method at all).